test/hmactest.c

```diff
@@ -133,6 +133,9 @@ static int test_hmac_run(void)
     int ret = 0;
 
     ctx = HMAC_CTX_new();
+    if (!ctx) {
+      return ret;
+    }
```

Comment:
Code style:
- no `{` `}` for single statement bodies
- we don't do `!ptr`, we use `ptr != NULL`
This should thus be:

```c
if (ctx == NULL)
    return 0;
```